src/neo-vm/Debugger.cs

```diff
@@ -64,22 +62,22 @@ public VMState StepOut()
             int c = engine.InvocationStack.Count;
             while (!engine.State.HasFlag(VMState.HALT) && !engine.State.HasFlag(VMState.FAULT) && !engine.State.HasFlag(VMState.BREAK) && engine.InvocationStack.Count >= c)
                 ExecuteAndCheckBreakPoints();
-            if (engine.State == VMState.NONE)
-                engine.State = VMState.BREAK;
```

Comment:
Noooo.... When the script is executed, it is possible to enter the `FAULT` state. We cannot directly rewrite it as `BREAK` here.

Reply:
I remove both lines, so is not rewrited his state

Reply:
But after `StepOut()` is finished, you need to change the state to `BREAK`.

Reply:
But is not possible to be `NONE`, so already is `FAULT`, `HALT` or `BREAK`

Reply:
L63
```c#
engine.State &= ~VMState.BREAK;
```

Reply:
I was looking in the wrong method, you are right xD

Reply:
But check this
https://github.com/neo-project/neo-vm/pull/185/files#diff-ee43c09353d045e18dc0046eb81d86dfR66

Reply:
One of my problems is that i can touch this line with ut https://codecov.io/gh/neo-project/neo-vm/src/master/src/neo-vm/Debugger.cs

Reply:
Don't understand.